// 8. Create a promise that resolves successfully or
//  rejects with a specific error, and handle the error using .catch.
console.lgo(ss'ddd)